Dockerfile

```diff
@@ -9,28 +23,26 @@ ENV APP_BUILD_TAG ${APP_BUILD_TAG}
 ARG APP_GIT_COMMIT
 ENV APP_GIT_COMMIT ${APP_GIT_COMMIT}
 
+ENV APPUID 1000
+
 ENV RAILS_ENV production
+ENV BUNDLE_APP_CONFIG="app/.bundle"
+
 ENV PUMA_PORT 3000
 EXPOSE $PUMA_PORT
 
-WORKDIR /app
-
-# ugrade bundler to 2.1.4
-RUN gem update bundler --no-document
-
-COPY . /app
+RUN addgroup -g $APPUID -S appgroup && \
+    adduser -u $APPUID -S appuser -G appgroup
 
-RUN bundle install --verbose --without="development test" --jobs 4 --retry 3
+RUN apk add tzdata postgresql-dev
```

Comment:
could we append `--update here, so it becomes `apk add --update` to get the latest list of packages?

Reply:
Also we might add `rm -rf /var/cache/apk/*` although not sure how much that shaves off the final package size here?

Reply:
Regarding the cache, I also had `--no-cache` here to the same effect. I must have dropped a commit instead of flatten it while rebasing.  This one saves 2MB as I recall. 

and 👍 for the --update. I'm a bit on two minds about this, as in one occasion it brought a broken updated µlibc. But on the other hand it usually get's rid of CVEs quite quickly, so it's worth have it.